internal/cli/cmd/plugin/build/build.go

```diff
@@ -214,21 +218,21 @@ func (o *option) prepareCueFiles() (func(), error) {
 	// managed the old way. This is technically possible & supported by CUE, thus we want to handle this case.
 	var restoreVendorDir func()
 	if _, err := os.Stat(o.vendorDirPath); err == nil {
-		if err := file.CopyDir(o.vendorDirPath, o.vendorDirBackupPath); err != nil {
-			return nil, fmt.Errorf("failed to backup original %s: %w", o.vendorDirPath, err)
+		if copyErr := file.CopyDir(o.vendorDirPath, o.vendorDirBackupPath); copyErr != nil {
```

Comment:
these renaming of the error variable are here to avoid the shadowing variable issue

Reply:
I don't think the shadowing causes an issue here though?

Reply:
no it doesn't, for this part it's more a good practice to avoid the shadowing variable. It avoid any confusion while reading the code.